src/shared/modules/cypher/boltUserHelper.js

```diff
@@ -18,31 +18,65 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-export function listUsersQuery () {
+export function listUsersQuery (is40 = false) {
+  if (is40) {
+    return 'SHOW USERS'
+  }
+
   return 'CALL dbms.security.listUsers'
 }
-export function listRolesQuery () {
-  return 'CALL dbms.security.listRoles YIELD role'
+
+export function listRolesQuery (is40 = false) {
+  if (is40) {
+    return 'SHOW ALL ROLES'
+  }
+
+  return 'CALL dbms.security.listRoles'
 }
-export function createDatabaseUser ({
-  username,
-  password,
-  forcePasswordChange
-}) {
+export function createDatabaseUser (
+  { username, password, forcePasswordChange },
+  is40 = false
+) {
+  if (is40) {
+    return `CREATE USER ${username} SET PASSWORD "${password}" CHANGE ${
```

Comment:
wdyt about using params when setting the password? 
I see you followed the pattern already in the existing code, but what happens if a user has a `"` in the password?